src/platform/windows/display_ram.cpp

```diff
@@ -268,6 +294,11 @@ capture_e display_ram_t::snapshot(::platf::img_t *img_base, std::chrono::millise
     return capture_e::timeout;
   }
 
+  // Now that we know the capture format, we can finish creating the image
+  if(complete_img(img, false)) {
+    return capture_e::error;
```

Comment:
This has the same issue as the hw encoder (see my other comment). Changing it to ok or timeout avoids the stream closing with Gravity Mark, but it seems that it can trigger the `complete_img()` error message for several frames concurrently.